### DISCUSSION TITLE
Discussion responses This is my general code discussion

### DISCUSSION THREAD

**Discussion by ThanThanSwe** [community/community#138410] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

 This is a general.
 
 Response #1.

**Comment by thanthanswe0310** [comment]
Response #2.

**Reply by ThanThanSwe** [reply]
here is my response #2.

**Reply by thanthanswe0310** [reply]
please revise your response #1.

**Reply by thanthanswe0310** [reply]
Response ##1

**Comment by ThanThanSwe** [comment]
here is a revised response @thanthanswe0310

**Comment by thanthanswe0310** [comment]
response ##2

**Reply by ThanThanSwe** [reply]
response ##2 needs to modified.

**Comment by thanthanswe0310** [comment]
response ###2.

**Comment by ThanThanSwe** [comment]
response ###2 needs to modified.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/138417#discussioncomment-10618052) for reference to why this discussion was closed and locked. Thanks!
